benchmarks/instruction_counts/definitions/standard.py

```diff
@@ -51,17 +51,79 @@
-                    setup=Setup.GENERIC.value,
-                ),
-            },
-        },
+        "Math": GroupedVariants(*parse_stmts(r"""
+            Python                                   | C++
+            ---------------------------------------- | ----------------------------------------
+            # @setup                                 | // @setup
+            torch.manual_seed(138_10_23)             | torch::manual_seed(1381023);
+            x = torch.rand((4, 4))                   | auto x = torch::rand({4, 4});
+            y_float = torch.ones((4, 4))             | auto y_float = torch::ones({4, 4});
+            y_vector = torch.ones((4, 1))            | auto y_vector = torch::ones({4, 1});
+            y_int = torch.ones(                      | auto y_int = torch::ones({4, 4}, at::kInt);
+                (4, 4), dtype=torch.int32)           |
+                                                     |
+            # @add                                   | // @add
+            x += 1.0                                 | x += 1;
+            x += y_float                             | x += y_float;
+            x += y_vector                            | x += y_vector;
+            x += y_int                               | x += y_int;
+            x + y_float                              | x + y_float;
+                                                     |
+            # @multiply                              | // @multiply
+            x *= 1.0                                 | x *= 1;
+            x *= y_float                             | x *= y_float;
+            x *= y_vector                            | x *= y_vector;
+            x *= y_int                               | x *= y_int;
+            x * y_float                              | x * y_float;
+                                                     |
+            # @equality                              | // @equality
+            x == y_float                             | x == y_float;
+            x == 1.0                                 | x == 1.0;
+        """)),
+
+        "Data movement": GroupedVariants(*parse_stmts(r"""
+            Python                                   | C++
+            ---------------------------------------- | ----------------------------------------
+            # @setup                                 | // @setup
+            x = torch.ones((4, 4))                   | auto x = torch::ones({4, 4});
+            y = torch.ones((4, 4))                   | auto y = torch::ones({4, 4});
+                                                     |
+            # @contiguous (trivial)                  | // @contiguous (trivial)
+            x.contiguous()                           | x.contiguous();
+                                                     |
+            # @contiguous (non-trivial)              | // @contiguous (non-trivial)
+            x.t().contiguous()                       | x.t().contiguous();
```

Comment:
This benchmarks `.t().contiguous()`, might be better to separate`.t()` into setup.

Reply:
Good point. I moved the transpose to setup, and also added a transpose benchmark. (Even though it's a degenerate case of `as_strided` it's a super important / common one.)